python/lsst/ts/tunablelaser/mock_server.py

```diff
@@ -881,9 +881,9 @@ def __init__(self):
         self.propagating = Power.OFF.value
         self.output_energy_level = Output.OFF.value
         if not self.scu:
-            self.configuration = NoSCU.NO_SCU.value
+            self.configuration = OpticalConfiguration.NO_SCU.value
```

Comment:
It's actually not necessary to use value anymore, you can just use NO_SCU directly. This was needed for python 3.10 compatibility. Please change this anywhere found in the code. Thanks!